projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.tsx

```diff
@@ -135,7 +134,11 @@ export class DotInputCalendarComponent {
     private emitValueChange(): void {
         this._valueChange.emit({
             name: this.name,
-            value: this.value
+            value: this.formattedValue()
         });
     }
+
+    private formattedValue(): string {
+        return this.value.length === 5 ? `${this.value}:00` : this.value;
```

Comment:
Are we sure this is safe enough?

Reply:
What is 5?

Reply:
possible values are `12:34:32` | `12:34` | `2000-02-02`. The time input remove the seconds when they are zero. is to force `HH:mm:ss` format